include/zephyr/bluetooth/mesh/brg_cfg.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg Bridge Configuration common header
+ * @ingroup bt_mesh
+ * @{
+ */
+
+#define BT_MESH_SUBNET_BRIDGE_DISABLED 0x00
+#define BT_MESH_SUBNET_BRIDGE_ENABLED  0x01
+
+/** Bridging Table state entry corresponding to a entry in the Bridging Table. */
+struct bridging_table_entry {
+	/** Allowed directions for the bridged traffic (or bridged traffic not allowed) */
+	uint32_t directions:8;
```

Comment:
Quite arguable approach to use bit fields in public API. How about bit field compatibility between platforms with different endians? Also, the sequence of fields within a data unit is not guaranteed by C standard. One compiler might pack them in one order, the others in another.